src/HtmlUtils.tsx

```diff
@@ -27,6 +27,10 @@ import _linkifyString from 'linkifyjs/string';
 import classNames from 'classnames';
 import EMOJIBASE_REGEX from 'emojibase-regex';
 import url from 'url';
+import katex from 'katex';
+import { AllHtmlEntities } from 'html-entities';
+import SettingsStore from './settings/SettingsStore';
+import cheerio from 'cheerio';
```

Comment:
Although `cheerio` is a dependency via some other dependency, it isn't specified in our `package.json`, which means that if its reverse-dependency ever stops depending on it, this will break.  So `cheerio` should be added to our `package.json` directly.

Reply:
done in 173d798